terraform/jenkins-controller.tf

```diff
@@ -207,3 +207,103 @@ resource "azurerm_key_vault_access_policy" "binary_cache_signing_key_jenkins_con
     "Get",
   ]
 }
+
+resource "azurerm_key_vault" "sigkv" {
+  name                = "ghaf-sig-kv-${local.ws}"
+  location            = azurerm_resource_group.infra.location
+  resource_group_name = azurerm_resource_group.infra.name
+  tenant_id           = data.azurerm_client_config.current.tenant_id
+  sku_name            = "standard"
+
+  # Access policy for the authenticated user
+  access_policy {
+    tenant_id = data.azurerm_client_config.current.tenant_id
+    object_id = data.azurerm_client_config.current.object_id
+
+    key_permissions = ["Get", "List", "Sign", "Verify"]
+    secret_permissions = ["Get", "List"]
+    certificate_permissions = ["Get", "List", "Create"]
+  }
+  
+  # Access policies
```

Comment:
As the earlier access_policy was "for the authenticated user", I assume this to be for non-authenticated. Maybe this comment too should spell it out.

Reply:
This access policy is for the Jenkins controller VM. The previous policy is required for the terraform to be able to create the certificates. I will clarify in the comment.